regolith/schemas.py

```diff
@@ -838,7 +847,10 @@
                                'type': 'integer'},
                            'end_month': {'required': False, 'type': 'string'},
                            'end_year': {'required': False, 'type': 'integer'},
-                           'location': {'required': True, 'type': 'string'},
+                           'group': {'required': False, 'type': 'string',
```

Comment:
Should we have a higher level of validation where we check that `fuzzy_finder` finds the linkages between collections? (eg someone puts in a bad group name).

Reply:
We will need that for this infrastructure to all work.  Because the "condition" is not just that ``group`` contains a ``string``, but that the ``string`` in ``group`` is in ``groups.yml``

This is all getting very complicated, and at some point not worth it maybe, but if we can do it, it will take a lot of pressure off the data-entry and bug chasing of data entry later, so will make the students much more happy!

Reply:
I think this can be handled by a separate validation system. It would not be hard to:
1. add something to the schema marking this as a cross-linked field and what it is cross linked to
1. Have a separate validation system which checks that the `fuzzy_finder` finds something for this, if not say something.